YUViewLib/src/statistics/StatisticsFileVTMBMS.cpp

```diff
@@ -191,6 +191,8 @@ void StatisticsFileVTMBMS::loadStatisticData(StatisticsData &statisticsData, int
   {
     statisticsData.setFrameIndex(poc);
 
+    std::unique_lock<std::mutex> lock(statisticsData.accessMutex);
```

Comment:
This does not feel right yet. @ChristianFeldmann what do you think? setFrameIndex a line above also uses the mutex and the other uses of the mutex are all handled by the class itself. Thats why it seems strange to put it like this. I think the csv statistic parser should have the same problem. we should check.

Reply:
First time using the review functionality, did it wrong somehow xD